src/definitions/modules/slider.less

```diff
@@ -110,11 +110,15 @@
   right: 0;
 }
 
-.ui.reversed.slider .inner .thumb {
+.ui.reversed.slider:not(.vertical) .inner .thumb {
   left: auto;
   right: 0;
 }
 
+.ui.reversed.vertical.slider .inner .thumb {
+    left: 0.03em;
+  }
```

Comment:
Please fix spacing so it matches the selector position 🙂

Reply:
I wonder how that happened! I also fixed the indentation to 2 spaces, my bad